lsp-protocol.el

```diff
@@ -290,6 +290,9 @@ See `-let' for a description of the destructuring mechanism."
                (rust-analyzer:CommandLink (:title :command) (:arguments :tooltip))
                (rust-analyzer:CommandLinkGroup (:commands) (:title)))
 
+(lsp-interface
+ (PrepareRenameDefaultBehaviorResult (:defaultBehavior) nil))
```

Comment:
nit: can you group this with the `lsp-interface` block that contains `Edit` or group this with the auto-generated block instead?
Since we will not doing auto generating anymore.